src/components/Answer.js

```diff
@@ -1,12 +1,16 @@
 import React from 'react'
 import PropTypes from 'prop-types';
 
+const handleMsg = (str) => {
```

Comment:
You can simplify this by rendering `str.toLowerCase()` directly and applying the css property `text-transform: capitalize` 😉

Reply:
Looks better, will change it!